mako/utils/transformers.cc

```diff
@@ -0,0 +1,114 @@
+// Adapted from https://github.com/vllm-project/vllm/blob/v0.2.7/vllm/model_executor/weight_utils.py
+// Copyright 2024 The Mako Authors
+// Copyright 2023 The vLLM team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "mako/utils/transformers.h"
+
+#include <vector>
+#include <iostream>
+#include <fstream>
+#include <tuple>
+#include <string>
+#include <filesystem>
+#include <algorithm>
+
+namespace fs = std::filesystem;
```

Comment:
The ``namespace fs`` is already aliased in ``mako/utils/transformers.h``. This line can be removed.